example/package.json

```diff
@@ -14,16 +14,14 @@
   },
   "dependencies": {
     "@react-native-community/async-storage": "^1.12.1",
-    "@react-native-community/netinfo": "^5.9.9",
     "babel-plugin-transform-inline-environment-variables": "^0.4.3",
     "react": "^17.0.1",
     "react-native": "0.63.4",
     "react-native-haptic-feedback": "^1.11.0",
     "react-native-linear-gradient": "^2.5.6",
     "react-native-permissions": "^3.0.0",
-    "react-native-spokestack": "4.0.0",
-    "react-native-video": "^5.1.0-alpha8",
-    "rn-fetch-blob": "^0.12.0"
+    "react-native-spokestack": "^5.0.0-beta.4",
```

Comment:
i assume this will be updated once rns hits 5.0.0?

Reply:
Yes, and the tray will be in beta until then.